.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: true
+contact_links:
+  - name: 🗣 Start a discussion on GitHub
+    url: https://github.com/google/iree/discussions
+    about: For topics that don't fit into a more specific category
+  - name: 📧 Send an email to our mailing list
+    url: https://groups.google.com/forum/#!forum/iree-discuss
+    about: For announcements and lower priority discussion
```

Comment:
I don't think we should say that the email list is for lower priority discussions *and* announcements. That doesn't make much sense to me. AFAIK we also never really had  discussion about using GitHub discussions, so now we're kind of just fragmented between even more channels. We could of course switch (e.g. make the email list for announcements only), but I don't think a PR is the right place to make that switch. So I would put the email list and discussions on equal footing. I actually didn't know we were using discussions at all...

Reply:
Yeah, agreed about iterating on this. I was referencing the existing wording at https://google.github.io/iree/#communication-channels and then trying to differentiate between GitHub Discussions and the mailing list. We could just list one or the other here. If someone is already on GitHub, maybe it's easier to just direct to the discussions instead of off to the mailing list.

Reply:
I can't think of a better description, so I just made them both say "For topics that don't fit into a more specific category". Any suggestions?

Reply:
I don't really understand what the "lower priority" was referring to regarding the mailing list. Where do "higher priority" discussions go? I think it's meant to be contrast with realtime discussion on Discord?

How about listing email list first and then GitHub discussion as a new alternative with similar scope? I'll attach a suggestion